docs/planning/sprintplanning.md

```diff
@@ -0,0 +1,66 @@
+# Knative Eventing sprint planning
+
+This document summarizes project planning mechanics for Knative Eventing releases. Knative Eventing release is planned every 6 weeks and will continue to be so. Current proposal is to align sprint with the release cycle of 6 weeks. Eventually this may change as we learn from the process.
+
+## How to manage Bugs?
+- Submit new bugs by creating a new issue of type “Report Bug”.
+- Please fill in as much detail as possible in the bug template and propose priority and severity
+- All new bugs will be labelled “triage/needstriage” and kind/bug as part of the template
+- Bugs will be triaged weekly (this could be modified based on incoming bugs rate). And upon triage will be labeled triage/accepted, or triage/rejected (and closed) with appropriate priority (priority/*) and severity (severity/*)
+- priority/p1 and priority/p2 bugs will be planned for next release cycle to keep low engineering debt.
+- priority/p0 is reserved for critical bugs that needs to be addressed immediately in current release at the cost of currently planned items.
+- WG lead will drive this and his/her decision will be final
+
+## How to manage Story/Feature?
+- Story/Feature typically spans 1-2 weeks for one engineer.
+To propose a new feature,  create a new issue of type “Feature Request”
+- Please fill in as much detail as possible in the Feature template and propose a release cycle
+- All new features will be labelled “kind/feature”, and “triage/needstriage”
+- All new proposals are discussed either in WG meeting, or on slack, or dedicated meetings if needed at any point scheduled by the proposer. 
+- By R-2w’s WG meeting all new proposals should be vetted and verified that they fit the [Planning criteria](#planning-criteria). Features that do not fit this criteria will not be considered in the current release planning cycle and hence early discussion is encouraged instead of waiting for R-2w.
```

Comment:
If this is a timeline, it's probably worth formatting these times to stand out (i.e. `**R-2w** (2 weeks before release)` (and then just bold for timing after that).

I'd find the timeline easier to read if it was called out into a specific section rather than integrated into the text. (Either as a table or as a separate list with each time point as a leading item with its own bullet.)